pa_matchbatch.sh

```diff
@@ -207,42 +210,46 @@ for currentItem in ${repoRevMap}; do
 	to_rev=$(echo -n ${currentItem} | cut -d'|' -f1)
 	repo_url=$(echo -n ${currentItem} | cut -d'|' -f2)
 	repo=$(parseRepoName ${repo_url})
-	foss_name=$(stripPrivateSuffix ${repo})
+	public_name=$(stripPrivateSuffix ${repo})
```

Comment:
`foss_name` did not seem correct -- these are foss repos generally, just private forks of them (there are no enterprise features in the private forks, for example)